openmdao/docs/openmdao_book/features/experimental/jax_explicitcomp_api.ipynb

```diff
@@ -0,0 +1,303 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.            self.add_input('z', val=np.zeros(2))
</pre><p>Is it necessary to use both <code>np</code> and <code>jnp</code> in this component definition? Users might find that confusing, or say "well I'm importing jnp anyway so I'll just use <code>jnp.zeros</code>."</p><p> <span class="ql-cursor">﻿</span></p><p>Could we override add_input and add_output here to allow jax arrays?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO/pull/3267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
It works either way.  I changed it to jnp.